curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/rotate.md

```diff
@@ -7,21 +7,15 @@ videoUrl:
 
 ## Description
 <section id='description'>
-Welcome to freeCodeCamp's HTML coding challenges. These will walk you through web development step-by-step.
-First, you'll start by building a simple web page using HTML. You can edit <code>code</code> in your <code>code editor</code>, which is embedded into this web page.
-Do you see the code in your code editor that says <code>&#60;h1&#62;Hello&#60;/h1&#62;</code>? That's an HTML <code>element</code>.
-Most HTML elements have an <code>opening tag</code> and a <code>closing tag</code>.
-Opening tags look like this:
-<code>&#60;h1&#62;</code>
-Closing tags look like this:
-<code>&#60;/h1&#62;</code>
-The only difference between opening and closing tags is the forward slash after the opening bracket of a closing tag.
-Each challenge has tests you can run at any time by clicking the "Run tests" button. When you pass all tests, you'll be prompted to submit your solution and go to the next coding challenge.
+In many cases we need to rotate image. In openCV you can do this way to matrix transformations.
+We use the function: <code>cv.rotate(src, dst, cv.ROTATE_90_CLOCKWISE)</code> to rotate on 90 degree where <code>src</code> is an input image,<code>dst</code> is destination one and <code>code</code> is a rotation code.
+
```

Comment:
In the previous challenges we described such parameters in general. 

see https://docs.opencv.org/master/d2/de8/group__core__array.html#ga4ad01c0978b0ce64baa246811deeac24 for the name of third argument.

Reply:
Please fix commas and whitespaces:
```
<code>cv.rotate(src, dst, rotateCode)</code>
```